### DISCUSSION TITLE
Show ImageType field as actual <img> tag on list and view templates

### DISCUSSION THREAD

**Discussion by dhait** [aminalaee/sqladmin#492] [Q&A]
How would you have an ImageType stored in the database display as the actual image on the list and view pages?

**Comment by aminalaee** [comment]
You need to override the list and view HTML templates as there's no way to know if you want to show <img> tags. And even if we added the option, there's the question of resizing and optimizations to show the image.

**Reply by dhait** [reply]
So would it be correct to say that the only feasible way to do this would be to use my own list/view templates for this model?
